.github/workflows/checks.yaml

```
@@ -388,17 +388,107 @@ jobs:
           file: ./Dockerfile
           push: false
 
+  prepare-sdk-platform-ref:
+    name: Prepare SDK platform ref
+    permissions:
+      contents: read
+    runs-on: ubuntu-22.04
+    outputs:
+      platform_ref: ${{ steps.compute.outputs.platform_ref }}
+      sdk_ref: ${{ steps.compute.outputs.sdk_ref }}
+    steps:
+      - name: Detect SDK backport branch
+        id: detect
+        shell: bash
+        run: |
+          BRANCH_NAME="${GITHUB_HEAD_REF:-${GITHUB_REF_NAME}}"
+          # Determine base branch of PR if available
+          BASE_REF="${GITHUB_BASE_REF:-}"
+          if [ -z "$BASE_REF" ] && [ -n "$GITHUB_EVENT_PATH" ] && [ -f "$GITHUB_EVENT_PATH" ]; then
+            BASE_REF=$(jq -r '.pull_request.base.ref // empty' "$GITHUB_EVENT_PATH" 2>/dev/null || true)
```

Without the `-e` option, jq will only fail if the query string is bad, so there isn't a need for the `|| true`. Also, I'd suggest leaving the error stream intact for debugging.

```suggestion
            BASE_REF=$(jq -r '.pull_request.base.ref // empty' "$GITHUB_EVENT_PATH")
```